modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -308,15 +315,14 @@ title: Example API Spec
      application/json:
        properties:
          firstName: string
-       examples:
-         someName: !include ex.raml
+       example: !include ex.yaml
 ----
 
-Further suppose that the NamedExample fragment also names the example, giving it the name `someFragmentName`:
+Further suppose that the fragment also names the example, giving it the name `someFragmentName`:
+
+.ex.yaml
 ----
-#%RAML 1.0 NamedExample
-someFragmentName:
-  firstName: Martin Gutierrez
+firstName: Martin Gutierrez
```

Comment:
if it's only the **first**name, then value should be `Martin` instead of `Martin Gutierrez`